doc/changelog.md

```diff
@@ -2,10 +2,72 @@
 
 <h3>New features</h3>
 
+* Catalyst users can now use Python control flow statements in their programs without having to
+  explicitly use the functions `cond`, `for_loop`, and `while_loop`!
+  [#235](https://github.com/PennyLaneAI/catalyst/pull/235)
+
+  A new experimental feature was added that automatically converts Python control flow statements
+  like `if`, `for`, and `while` into their equivalent functional forms provided by Catalyst.
+  The feature is based on the AutoGraph module from TensorFlow, and requires a working TensorFlow
+  installation.
+
+  This example shows how existing control flow in Catalyst:
+
+  ```python
+  @qjit
+  @qml.qnode(qml.device("lightning.qubit", wires=n))
+  def f(x):

```

Comment:
very minor, but I recommend swapping the two examples :) Show users how easy it is with this change, and then show what it corresponds to in the previous UI.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/235/commits/338ca376e9caa46a69118325db4cb0621f5e4f05